tests/conftest.py

```diff
@@ -181,43 +133,71 @@ def _inner(workflow_id, opts=None):
         output = SimpleNamespace()
 
         try:
-            cylc_reinstall(options, workflow_id)
+            if parser_reqd:
+                script(parser, options, str(srcpath))
+            else:
+                script(options, str(srcpath))
             output.ret = 0
             output.exc = ''
         except Exception as exc:
             output.ret = 1
             output.exc = exc
+
         output.logging = '\n'.join([i.message for i in caplog.records])
         output.out, output.err = capsys.readouterr()
+
         return output
     return _inner
 
 
 @pytest.fixture
 def cylc_install_cli(capsys, caplog):
-    return _cylc_install_cli(capsys, caplog)
+    return _cylc_cli(
+        capsys, caplog,
+        cylc_install, install_gop
+    )
 
 
 @pytest.fixture(scope='module')
 def mod_cylc_install_cli(mod_capsys, mod_caplog):
-    return _cylc_install_cli(mod_capsys, mod_caplog)
+    return _cylc_cli(
+        mod_capsys, mod_caplog,
+        cylc_install, install_gop
+    )
```

Comment:
Maybe do this more similarly to the Cylc integration tests.

Reply:
This might be too long ago, and I'm not terribly familiar with the Cylc Integration tests... but...

My impression is that they largely revolve around running flows which until recently were written to the flow directory.

Here most of the tests revolve around the install process, so I'm not sure that the same approach makes sense.

Reply:
I think my point was that this implementation differs from the one you are currently adding in cylc-flow with the `cylc vr` pull request. It would make sense to do it the same way for both.